test/bin/stop_api_test_server.sh

```diff
@@ -53,7 +53,7 @@ shutdown_osmt_app(){
     echo "INFO: Application ${OSMT_APP_CLASS} found PID ${app_pid}."
   fi
 
-  kill -SIGINT "${app_pid}"
+  kill -9 "${app_pid}"
```

Comment:
kill -9 is very messy. Please see the 1st answer here:
* https://stackoverflow.com/questions/43724467/what-is-the-difference-between-kill-and-kill-9. kill -SIGINT is much more graceful, and allows commits to complete / logs to be written / flies to be closed. kill -9 is too heavy for this. Please put -SIGINT back, or let's talk about what other approach you need here?